### PR TITLE
iOS Support

### PR DESCRIPTION
**Pull request by omefire**
iOS Support

### CODE REVIEW

**Comment by omefire** [comment]
@nikhilkh to review.

File: lib/livereload.js [hunk 1]
```diff
@@ -73,8 +73,12 @@ function StartServer() {
             }
 
             // After a file changes, first run prepare, then reload.
-            Q().then(function() {
-                self.context.cordova.prepare();
+            return self.context.cordova.raw.prepare().then(function() {
+                // Remove CSP directive from HTML
+                var CSPRemover = require('./utils/CSPRemover');
```
Comment (nikhilkh) [inline lib/livereload.js]: Nit: consider defining CSPRemover further up as on every change you do not need to run 'require'

File: lib/utils/CSPRemover.js [hunk 2]
```diff
@@ -2,28 +2,36 @@ var cheerio = require('cheerio');
...
 
         var $ = cheerio.load(self.indexHTMLContent);
         $("meta[http-equiv=Content-Security-Policy]").remove(); // what if csp is in lower character ? 	
 
-        var newHtmlContent = $.html(); // HTML content with CSP directives removed	
-        fs.writeFile(self.indexHTMLPage, newHtmlContent, function(err) {
+        var newHtmlContent = $.html(); // HTML content with CSP directives removed
+
+        // Override file with new content by first performing a deletion followed by creating a new file because of a weird bug 
+        //   ... where sometimes fs.writeFile(..) doesn't override file content with new text on ios
```
Comment (nikhilkh) [inline lib/utils/CSPRemover.js]: Is this an issue in iOS or OSX host operating system?

**Comment by omefire** [comment]
Merged:
1- https://github.com/omefire/cordova-plugin-livereload/commit/a0e3638feb1f10f133e1eab97ef562877f9a16b9

2- https://github.com/omefire/cordova-plugin-livereload/commit/0e2c9d1248ceee301993ec6afb70dc7f9e3bba00

**Pull request closed without merging**